ui/src/app/kvm/components/KVMActionFormWrapper/ComposeForm/InterfacesTable/SubnetSelect/SubnetSelect.test.tsx

```diff
@@ -0,0 +1,172 @@
+import { act } from "react-dom/test-utils";
+import { mount } from "enzyme";
+import React from "react";
+import { MemoryRouter, Route } from "react-router-dom";
+import { Provider } from "react-redux";
+import configureStore, { MockStoreEnhanced } from "redux-mock-store";
+
+import type { MenuLink } from "./SubnetSelect";
+import type { Pod } from "app/store/pod/types";
+import {
+  domainState as domainStateFactory,
+  fabricState as fabricStateFactory,
+  generalState as generalStateFactory,
+  podDetails as podDetailsFactory,
+  podState as podStateFactory,
+  podStatus as podStatusFactory,
+  powerType as powerTypeFactory,
+  powerTypesState as powerTypesStateFactory,
+  resourcePoolState as resourcePoolStateFactory,
+  rootState as rootStateFactory,
+  space as spaceFactory,
+  spaceState as spaceStateFactory,
+  subnet as subnetFactory,
+  subnetState as subnetStateFactory,
+  vlanState as vlanStateFactory,
+  zoneState as zoneStateFactory,
+} from "testing/factories";
+import ComposeForm from "../../ComposeForm";
+
+const mockStore = configureStore();
+
+const generateWrapper = (store: MockStoreEnhanced, pod: Pod) =>
+  mount(
+    <Provider store={store}>
+      <MemoryRouter
+        initialEntries={[{ pathname: `/kvm/${pod.id}`, key: "testKey" }]}
+      >
+        <Route
+          exact
+          path="/kvm/:id"
+          component={() => <ComposeForm setSelectedAction={jest.fn()} />}
+        />
+      </MemoryRouter>
+    </Provider>
+  );
+
+describe("SubnetSelect", () => {
+  let initialState = rootStateFactory();
```

Comment:
Maybe this should be `let initialState: RootState;`?

Reply:
Good call. I just copy+pasted this from another file so I'll update it there too.